tests/configs/docker_registry/k8s_deployments.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations: {app.kubernetes.io/managed-by: kubetools}
+  labels: {kubetools/name: generic-app, kubetools/project_name: generic-app, kubetools/role: app}
+  name: generic-app
+spec:
+  replicas: 1
+  revisionHistoryLimit: 5
+  selector:
+    matchLabels: {kubetools/name: generic-app, kubetools/project_name: generic-app,
+      kubetools/role: app}
+  template:
+    metadata:
+      labels: {kubetools/name: generic-app, kubetools/project_name: generic-app, kubetools/role: app}
+    spec:
+      containers:
+      - command: [generic-command]
+        containerContext: generic-context
```

Comment:
Is this supposed to be in there?

Reply:
https://github.com/EDITD/kubetools/pull/141/commits/b69af2413f6bf45f793c6462ce127b2478927dd2